src/consensus.rs

```diff
@@ -371,10 +371,17 @@ impl Consensus {
 }
 
 fn handle_committed_entries(
-    entries: Vec<Entry>,
+    mut entries: Vec<Entry>,
     state: &ConsensusStateRef,
     raw_node: &mut RawNode<ConsensusStateRef>,
 ) -> raft::Result<()> {
+    let last_applied = state.persistent.read().last_applied_entry();
+    if let Some(last_applied) = last_applied {
```

Comment:
Maybe we can have a `log::debug` here explaining that we are filtering the entry up to last_applied.

Reply:
Then we will always see this on restart if we have a debug log level, I am not so sure it is needed

Reply:
Alright makes sense 👍